front-end/README.md

```diff
@@ -1,70 +1 @@
-# Getting Started with Create React App
```

Comment:
I'd take out the README changes, we can make a comprehensive README in the root of the repo eventually but I'd just leave this one for now.

Reply:
yes, we'll update, this is just a placeholder